hibernate-core/src/main/java/org/hibernate/engine/internal/StatefulPersistenceContext.java

```diff
@@ -1935,11 +1917,12 @@ public void removeSharedNaturalIdCrossReference(EntityPersister persister, Seria
 			final Object naturalIdCacheKey = naturalIdCacheAccessStrategy.generateCacheKey( naturalIdValues, persister, session );
 			naturalIdCacheAccessStrategy.evict( naturalIdCacheKey );
 
-//			if ( sessionCachedNaturalIdValues != null
-//					&& !Arrays.equals( sessionCachedNaturalIdValues, deletedNaturalIdValues ) ) {
-//				final NaturalIdCacheKey sessionNaturalIdCacheKey = new NaturalIdCacheKey( sessionCachedNaturalIdValues, persister, session );
-//				naturalIdCacheAccessStrategy.evict( sessionNaturalIdCacheKey );
-//			}
+			// if ( sessionCachedNaturalIdValues != null
```

Comment:
this reformatting should probably not be applied?

Please never reformat existing code, unless it's the primary and exclusive goal of the patch.

Reply:
Seems I missed that, I'll undo that commit block somehow. Thanks for the advice